app/ore/project/io/PluginFileData.scala

```diff
@@ -0,0 +1,184 @@
+package ore.project.io
+
+import java.io.BufferedReader
+
+import com.google.gson.JsonParser
+import com.google.gson.stream.JsonReader
+import models.project.{Tag, TagColors}
+import ore.project.Dependency
+import org.spongepowered.plugin.meta.McModInfo
+import scala.collection.JavaConverters._
+
+import scala.collection.mutable.ArrayBuffer
+
+/**
+  * The metadata within a [[PluginFile]]
+  *
+  * @author phase
+  * @param data the data within a [[PluginFile]]
+  */
+class PluginFileData(data: Seq[DataValue[_]]) {
```

Comment:
Can we use scala naming conventions for this class? `x` instead of `getX`

Reply:
:relaxed: my Java is showing